Header  header
float32 x
float32 y
float32 min_distance
